libr/anal/var.c

```diff
@@ -359,6 +411,7 @@ R_API RAnalVar *r_anal_var_get(RAnal *a, ut64 addr, char kind, int scope, int de
 	av->name = vt.name? strdup (vt.name): strdup ("unkown_var");
 	av->size = vt.size;
 	av->type = vt.type? strdup (vt.type): strdup ("unkown_type");
+	av->regname = vt.regname? strdup (vt.regname): strdup ("unkown_regname");
```

Comment:
Put the ternary iperator inside the strdup call. Dont call strdup twice qhen its just one arg that changes